zigate/core.py

```diff
@@ -1254,19 +1255,19 @@ def _neighbours_table(self, addr, nodes=None):
         while index < entries:
             r = self.lqi_request(addr, index, True)
             if not r:
-                LOGGER.error('Failed to build neighbours table')
-                return
+                LOGGER.error('Failed to build neighbours table: addr=%s, index=%s', addr, index)
+                return []
```

Comment:
Return an empty list instead of `None` to avoid `'NoneType' object is not subscriptable` errors.

Reply:
Outdated.